app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md

```diff
@@ -66,7 +86,7 @@ If you’re not a UK citizen, you are unlikely to get help funding your training
 
 ## Check your eligibility for a scholarship or bursary
```

Comment:
We've changed the order of these around the site so it is `bursary or scholarship` as there are more bursaries available. Worth doing on this page whilst we are making updates?